runtime/src/lib.rs

```diff
@@ -593,12 +587,12 @@ impl pallet_authorship::Config for Runtime {
 // details and https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=e6490da3bfb28f69c7f6e6393ec6bb0c
 // for the calculation
 // TODO: set back to Perbill::from_percent(3) as soon as the bug above is fixed.
-const THREE_PERCENT_INFLATION: Perbill = Perbill::from_parts(29_559_999);
+//const THREE_PERCENT_INFLATION: Perbill = Perbill::from_parts(29_559_999);
 
-const REWARD_CURVE: PiecewiseLinear<'static> = PiecewiseLinear {
-	points: &[(Perbill::from_percent(0), THREE_PERCENT_INFLATION)],
-	maximum: THREE_PERCENT_INFLATION,
-};
+//const REWARD_CURVE: PiecewiseLinear<'static> = PiecewiseLinear {
+//	points: &[(Perbill::from_percent(0), THREE_PERCENT_INFLATION)],
+//	maximum: THREE_PERCENT_INFLATION,
+//};
```

Comment:
we are not using any of this constants anymore? Is the bug above not relevant anymore?

Reply:
that is for staking only. we have to clean up runtime

Reply:
Yep. I started to clean up unused imports and so. Hope I was not too Mr. Proper :)